src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/SaveTemplateModal.jsx

```diff
@@ -22,14 +22,14 @@ const sortNames = (a, b) => a.toLowerCase().localeCompare(b.toLowerCase());
 function Footer(props) {
   return (
```

Comment:
Is the save template modal just supposed to say "Modal" in the title?
![image](https://user-images.githubusercontent.com/8951708/70640263-508a3000-1c01-11ea-9579-d48b6cc1dd53.png)

Reply:
Well in that is what it says, hard coded on line 209

Reply:
Fixed in 423cddb16d369e97668b7c619338e4f342fe4d11